### PR TITLE
Handling encoding parameter errors

### PR DESCRIPTION
**Pull request by aboba**
Work in progress (not ready to merge).  Need to incorporate feedback from the virtual interim.

Fix for Issue https://github.com/w3c/webrtc-pc/issues/763

### CODE REVIEW

File: webrtc.html [hunk 1]
```diff
@@ -11702,6 +11705,19 @@ <h2 id="sec.idp-error-handling">IdP Error Handling</h2>
                 <td>The IdP token is invalid.</td>
               </tr>
               <tr>
+                <td><dfn><code>invalid-codecs</code></dfn></td>
+                <td>Invalid or unavailable <code>codecs[]</code>
```
Comment (taylor-b) [inline webrtc.html]: May want to restrict this to just "unavailable", since "invalid" is already covered by InvalidModificationError.

**Comment by fluffy** [comment]
This LGTM and like to see the stuff from call added in.

File: webrtc.html (hunk 1 above)
Comment (aboba) [inline webrtc.html]: What if setParameters() attempts to modify an RTCRtpCodecParameters dictionary entry in parameters.codecs[]?  Would this be covered by InvalidModificationError ?

Comment (taylor-b) [inline webrtc.html]: I thought it would be. Or at least that was our intention.

**Comment by stefhak** [comment]
Good enough to merge now?

**Comment by stefhak** [comment]
@aboba what's the status of this one now?

**Comment by aboba** [comment]
@stefhak I have closed the original issue, since we now have more specific Issues filed for each aspect of the problem. So this PR is now moot.

**Pull request closed without merging**